css/index.css

```diff
@@ -138,9 +139,112 @@ footer nav {
     align-items: center;
     padding: 20px 2%;
     font-size: 14px;
+    font-weight: bold;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+.navigation-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;    
+}
+.nav-bar a {
+    text-decoration: none;
+    padding: 10px;
+    color: black; 
+    display: block;
+
+}
+.nav-bar {
+    display: flex;
+}
+.nav-bar a:hover {
+    color:blue;
+    transform: rotate(10deg);
+}
+.lambda-logo {
+    padding: 15px;
+    margin-top: 10px;
+}
+.lambda-logo h1::first-letter{
+    font-size: 45px;
+    padding:0 10px;
+    background-color: black;
+    color: white;
+    text-align: center;
+}
+.lambda-logo h1:before {
+    content: '\03BB';
+     
+}
```

Comment:
Nice job using the parent styles to cascade down to the parent and only changing the color.  Using !important should only be used as a last resort, it breaks the cascading order and will cause a lot of debugging issues if used a lot.